.vscode/launch.json

```diff
@@ -1,31 +0,0 @@
-{
```

Comment:
Not a problem but personally I do drive by fixes like these to be off on their own PR?

Reply:
Yeah that's fair, I'll pull it out

Reply:
https://github.com/DFE-Digital/get-into-teaching-api/pull/5